src/app/app-layout/navbar/navbar.component.ts

```diff
@@ -43,27 +59,6 @@ export class NavbarComponent {
       filter(screen => screen !== ('sm' || 'md')),
       tap(() => this.appLayoutStore.closeDropdownMenu()),
     )
-
-    this.issuer$ = this.issuerService.issuerWithStatus$
-    this.navbarScreenLinks.push(
-      {
-        title: 'Admin',
-        routerLink: '/admin',
-        showItem: combineLatest([
-          this.isLoggedIn$,
-          this.sessionQuery.address$,
-          this.issuerService.issuer$
-        ]).pipe(
-          map(([isLoggedIn, address, issuer]) => {
-            if (!isLoggedIn || !address) {
-              return false
-            }
-
-            return address === issuer.owner
-          })
-        )
-      }
-    )
```

Comment:
the constructor will be run once when the component is rendered. consider the case when you are not logged, then you log in, you become an admin, and with this logic, the admin tab won't be visible until you refresh the entire page. 

consider not to use static logic and behave like the entire app is changing dynamically (as it actually does).